tests/readmes/test_c2d_flow.py

```diff
@@ -21,7 +23,35 @@
 
 
 @pytest.mark.integration
-def test_c2d_flow_readme():
+@pytest.mark.parametrize(
+    "dataset_name,dataset_url,algorithm_name,algorithm_url,algorithm_docker_tag",
```

Comment:
I think just one sample can be tested. The others maybe as part of the "slow" marker (see #734)

Reply:
Thanks for reviewing.  Respectfully, I disagree. The primary purpose of this test is to ensure the c2d READMEs work as intended. Therefore, they should mirror the README as closely as possible.

Reply:
yep :) exactly. Looks good.